python/src/trezorlib/cli/device.py

```diff
@@ -295,7 +302,7 @@ def unlock_bootloader(client: "TrezorClient") -> str:
 
 
 @cli.command()
-@click.argument("enable", type=ChoiceType({"on": True, "off": False}), required=False)
+@click.argument("enable", type=ChoiceType({"on": True, "off": False}))
```

Comment:
`required` defaults to `False` for _options_
not for arguments. this change is not actually a no-op